package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@team23/eslint-config-team23-vue",
+  "version": "1.0.0",
+  "description": "A set of eslint rules used by TEAM23 for Vue 3 projects",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "lint": "eslint test/* --no-ignore"
+  },
+  "keywords": [
+    "eslint",
+    "Vue 3",
+    "TEAM23"
+  ],
+  "author": "TEAM23 GmbH",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/team23/eslint-config-team23-vue"
+  },
+  "devDependencies": {
+    "eslint": "^8.47.0",
+    "eslint-plugin-vue": "^9.17.0"
```

Comment:
**thought**: not sure if is possible to instead of having this package as a dev dep and a peer dep, to have this as a dependency. Maybe you can try that?

```
dependencies: {
    "eslint-plugin-vue": "^9.17.0"
}
```